README.md

```diff
@@ -1 +1,55 @@
-# OMA-Bokkstore
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# OMA Bookstore
+## Description
+The Bookstore is a website similar to the "Awesome Books" website built in the previous module. You will create an MVP version of it that allows you to:
+
+- Display a list of books.
+- Add a book.
+- Remove a selected book.
+
+I build this application using React and Redux togethar.
+
+## Site Screenshots
+
+![screenshot](./Screenshot-1.png)
+
+
+## Built With
+
+- React.
+- HTML.
+- CSS.
+- Webpack.
+- JavaScript.
+- Linters.
+
+## Install
+
+- Install node.js from [HERE](https://nodejs.org/en/).
+
+## Test
+
+- `Npm run test`
+  
+## Deployment
+
+1. Clone the project using git-bash or Githup Desktop.
+2. Open the project folder with VSCode or any Editor.
+3. Open terminal and navigate to the project folder.
+4. Type `npm install`.
+5. Type `npm start`.
```

Comment:
- [ ] I think you can change the section title from `Deployment `to `Setup ` to make it meaningful.